astropy/modeling/utils.py

```diff
@@ -484,3 +484,59 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image):
```

Comment:
@embray - I don't think it's a good idea to have all model rendering code live on model classes.
At the moment `Model.__call__` gets `x`, `y` passed in ... here the `x` and `y` grids are computed on the fly from the input image and bounding box.
Of course it would be possible to introduce several cases and options into `Model.__call__`, but I think it would pretty soon lead to a very complex API.

Butter to implement some things as standalone functions for now, and to re-expose it via `Model` methods where appropriate. This also makes it easier to re-use the method @patti implements here (and more will come in the coming weeks) if the computations are separated from the framework at the low level.

If you don't like the function in this file, maybe a new `render.py` files in `astropy/modeling` would be OK with you?